domain-models-runtime/src/main/java/org/folio/rest/tools/utils/NetworkUtils.java

```diff
@@ -16,6 +16,10 @@
  */
 public class NetworkUtils {
 
+  /**
+ *  ThreadLocalRandom has a Random instance per thread and safeguards against contention.
+ */
+ @SuppressWarnings("all")
```

Comment:
If @SuppressWarnings("all") is replaced by `@SuppressWarnings("java:S2245")` other warnings will still be flagged when we introduce other issues in the code in future.